libraries/kitsu/library/src/main/java/com/chesire/nekome/kitsu/library/KitsuLibraryEntityMapper.kt

```diff
@@ -0,0 +1,48 @@
+package com.chesire.nekome.kitsu.library
+
+import com.chesire.nekome.core.EntityMapper
+import com.chesire.nekome.core.models.ImageModel
+import com.chesire.nekome.kitsu.library.entity.DataEntity
+import com.chesire.nekome.kitsu.library.entity.IncludedEntity
+import com.chesire.nekome.kitsu.library.entity.KitsuLibraryEntity
+import com.chesire.nekome.library.api.LibraryEntity
+import javax.inject.Inject
+
+/**
+ * Provides ability to map instances of [KitsuLibraryEntity] into [LibraryEntity].
+ */
+class KitsuLibraryEntityMapper @Inject constructor() :
```

Comment:
Does this need to be an interface, or could it just be a simple class defined in the Kitsu layer? Might make more sense and then less code needed to create it